lib/erl/src/thrift_processor.erl

```diff
@@ -19,13 +19,35 @@
 
 -module(thrift_processor).
 
--export([init/1]).
+-export([init/1, process/1]).
 
 -include("thrift_constants.hrl").
 -include("thrift_protocol.hrl").
 
 -record(thrift_processor, {handler, protocol, service}).
 
+% Instead of a loop, do a single-shot processing of a packet
+% allows user to control how the bytes are sent and received
+% see: [https://github.com/apache/thrift/blob/master/lib/rb/lib/thrift/server/mongrel_http_server.rb]
```

Comment:
Please link to a specific commit instead of master (press `y` while on that page) so comments like this survive any future refactor/reorganization.